effect/jvm/src/test/scala/scalaz/effect/generator/GenIO.scala

```diff
@@ -0,0 +1,122 @@
+package scalaz.effect.generator
+
+import org.scalacheck._
+import scalaz.effect.{ ExitResult, IO, RTS }
+
+trait GenIO extends RTS {
```

Comment:
Do we need to extend `RTS`? May be best to let test suites do that themselves.

Reply:
oou, it was just a for debugging purpose.... removed